### ISSUE TITLE
Allow changing of lightpost intensity in in-game map editor.

### ISSUE THREAD

**Issue by tovl** [OpenRA/OpenRA#18407]
After playing around a bit in the editor with the new lighting mechanics (which are absolutely awesome) it occurred to me that it would be very useful to be able to dynamically adjust the intensity of the lightpost actors with a slider, just like is currently possible for the facing and health of units. That would make it really easy to fine-tune the scene from an artistic viewpoint without having to add a new custom actor to the map rules for every slight variation you might want to have (and then going back into the game to check the results, then going out of the game to adjust, etc...).

I think having just a slider for intensity would be enough for most uses. Adding a new RGB-lightpost to the existing roster with separate sliders for red, green and blue for the really custom work would be the cherry on top.

**Comment by pchote** [comment]
Good idea. We could expose inits for both range and tint with only minor trait changes (the editor ui may be a bit trickier - an actual colour picker that generates a single LightpostColorInit might be nicer than 3 separate sliders and inits).
